graphql-cli/src/main/java/io/ballerina/graphql/cmd/GraphqlCmd.java

```diff
@@ -65,11 +75,9 @@
  * Main class to implement "graphql" command for Ballerina.
  * Commands for Client and SDL Schema file generation.
  */
-@CommandLine.Command(
-        name = "graphql",
+@CommandLine.Command(name = "graphql",
         description = "Generates Ballerina clients for GraphQL queries with GraphQL SDL and " +
-                "SDL schema for the given Ballerina GraphQL service."
-)
+                "SDL schema for the given Ballerina GraphQL service.")
```

Comment:
We need to update the descriptions with new feature details. check other places as well.